specs/darwin/running_apps.table

```diff
@@ -1,6 +1,7 @@
 table_name("running_apps")
 
-description("Returns the PID of the application currently in focus on MacOS.")
+description("Returns a list of running apps with their pids, bundle \
+  identifiers and whether or not they are currently in focus.")
```

Comment:
Looking at some of the other descriptions, I think something along the lines of "OS X applications currently running on the host system" would fit in better (that's kind of a hybrid of the `apps` and `processes` tables).